test/transcript/TranscriptEvent.test.ts

```diff
@@ -263,4 +264,36 @@ describe('TranscriptEvent', () => {
     expect(actualEvents[0].results[0].alternatives[0].entities[1].type).to.be.undefined;
     expect(actualEvents[0].results[0].alternatives[0].entities[1].confidence).to.eql(0);
   });
+
+  it('handles one transcript event of transcript type with language identifications', async () => {
+    const event = makeSdkTranscriptWithLanguageIdentifications();
+    const data = SdkTranscriptFrame.encode(makeSdkTranscriptFrame([event])).finish();
+    logBase64FromUint8Array(data);
+    // The steps above generates data below passed directly as test input
+    const dataMessage = makeTranscriptDataMessageFrom(
+        TRANSCRIPT_EVENT_TEST_VECTORS.TRANSCRIPT_LANGUAGE_IDENTIFICATIONS
+    );
+    const actualEvents = TranscriptEventConverter.from(dataMessage);
+    expect(actualEvents.length).to.equal(1);
+    if (!(actualEvents[0] instanceof Transcript)) {
+      assert.fail();
+      return;
+    }
+    expect(actualEvents[0].results.length).to.eql(1);
+    expect(actualEvents[0].results[0].alternatives[0].items.length).to.eql(3);
```

Comment:
Why not just assert that the decoded event is (deeply) equal to `event`?

Reply:
Updated assertions to use `event` instead of hard-coded values